plugins/action_change_status/src/changeStatus.ts

```diff
@@ -15,6 +15,14 @@ async function changeStatus({
       : parameters?.isReview || applicationData?.action_payload?.trigger_payload?.table === 'review'
   const newStatus = parameters?.newStatus
 
+  console.log(`Changing status of ${isReview ? 'Review' : 'Application'} to ${newStatus}`)
+
+  if (!newStatus)
+    return {
+      status: ActionQueueStatus.Fail,
+      error_log: 'Missing property: "newStatus"',
+    }
```

Comment:
I actually want to implement a more comprehensive 'required parameter' checker for all the actions. The required parameters are defined in the "plugin.json", so we just need a generic method to check each one right at the top of each action.

But this is a good start for now.